class.jetpack.php

```diff
@@ -5120,11 +5120,11 @@ public function alert_auto_ssl_fail() {
 	/**
 	 * Returns the Jetpack XML-RPC API
 	 *
+	 * @deprecated 8.0 Use Connection_Manager instead.
 	 * @return string
 	 */
 	public static function xmlrpc_api_url() {
-		$base = preg_replace( '#(https?://[^?/]+)(/?.*)?$#', '\\1', JETPACK__API_BASE );
-		return untrailingslashit( $base ) . '/xmlrpc.php';
+		return self::connection()->xmlrpc_api_url();
```

Comment:
Do we want to throw `_deprecated_function` here?

Reply:
Per Slack, we can throw the notice and update previous usages in a follow-up PR.